app/views/checkbox_criteria/_criterion_editor.html.erb

```diff
@@ -39,6 +39,14 @@
       <%= f.text_field :max_mark %>
     </span>
     <br>
+    <span id="<%= "criterion_inputs_#{criterion.id}" %>">
+      <% selected_files = criterion.criteria_assignment_files_joins.exists? ? criterion.criteria_assignment_files_joins.map { |join| join.assignment_file }: [] %>
```

Comment:
The check is unnecessary. Calling `map` on `criterion.criteria_assignment_files_joins` even when it's empty should be fine.